README.md

```diff
@@ -3,7 +3,9 @@
 
 `conda activate aya`
 
+Install `poetry` as described in https://python-poetry.org/docs/#installation and run `poetry install`
```

Comment:
@AMR-KELEG I am not sure how OK would you be with moving to Poetry (which may make conda not necessary in our case) but I believe there are advantages to it, especially if the CLI script might end up being an installable package one day.

Reply:
Hi Marek,

I am fine with using `poetry`. I have not used it before, but the instructions you left seem to be enough 👍🏽 
Thanks for the suggestion 😄